packages/conferer/src/Conferer/FromConfig/Internal.hs

```diff
@@ -346,7 +350,7 @@ missingRequiredKeys keys =
 -- | Exception to show that the provided default had the wrong type, this is usually a
 -- programmer error and a user that configures the library can not do much to fix it.
```

Comment:
defaults* ?, also, isn't this what's being removed in the PR?

Reply:
true, I'll remove all of this, I wasn't sure if that was a good idea but I now think that leaving it doesn't make much sense